pom.xml

```diff
@@ -151,39 +151,37 @@
 					</execution>
```

Comment:
@joaofreitas-qubit let's just checkout this file directly from master so it stops being seen as a change when it's only spacing.